### ISSUE TITLE
32 Bit vs 64 bit

### ISSUE THREAD

**Issue by charles-peter** [enthought/pyql#87]
I'm trying to build my setup.py right and running into a few a issues. From the error I get I think its something with 32vs64 bit compilers. I compiled QL in VC 2008 express which only allowed me to compile in 32. Okay so I lose some speed nbd. But my machine and several other programs I am using are 64 bit. I build the def file with 64 bit MinGW nm. Then when I run the setup.py build with the MS VC++ for python I can get two diffrent errors depending on wether its the 32vs64 bit compilers. With the 32 bit compiler I get "module machine type 'x86' conflictrs with target machine type 'x64'". Then with the 64 bit compiler I get 2 linker errors akin to what I was getting when I built QL with VS2013. Im guessing theres is a conflict somewhere down the line but was wondering if you guys ever had problems with a 32 bit build on 64 bit machines before?

**Comment by dpinte** [comment]
@charles-peter Yes, there is some hardcoded flags that need to be tweaked (and/or removed). For example: `/machine:I386` which is for 32bit systems only (https://github.com/enthought/pyql/blob/master/setup.py#L97). I think this is the only one. PR's are welcome.
